### PR TITLE
Add `scarb cairo-run` tests

### LINKED ISSUE #1089: Write tests for `scarb cairo-run` program arguments
We are lacking tests for this argument:

https://github.com/software-mansion/scarb/blob/e7fc484c79b713bebd6fda5fa0bc5a977e988232/extensions/scarb-cairo-run/src/main.rs#L40-L46

### PR DESCRIPTION
**Pull request by tomek0123456789**
Closes #1089 

Please let me know if I've missed any tests cases, thanks!

### CODE REVIEW

File: extensions/scarb-cairo-run/tests/arguments.rs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        .build(t);
+}
+
+#[test]
+fn valid_number_of_args() {
+    let t = TempDir::new().unwrap();
+    setup_fib_three_felt_args(&t);
+
+    Command::new(cargo_bin("scarb"))
+        .env("SCARB_TARGET_DIR", t.path())
```
Comment (maciektr) [inline extensions/scarb-cairo-run/tests/arguments.rs:31]: I don't think you need it here. It has been added to tests that run on projects from `examples` directory, as in that case the `target` dir is shared by all the tests (e.g. `examples/hello_world/target` - this path does not change at all!). Here you setup a temp dir for a test project each time, so I think you should be good to go. 
While at it - would you mind adding a comment in the other tests file explaining why has this been done?

File: extensions/scarb-cairo-run/tests/arguments.rs [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+#[test]
+fn invalid_number_of_args() {
+    let t = TempDir::new().unwrap();
+    setup_fib_three_felt_args(&t);
+
+    Scarb::quick_snapbox()
+        .arg("cairo-run")
+        .arg("--")
+        .arg(r#"[2, 1, 3, 7]"#)
```
Comment (maciektr) [inline extensions/scarb-cairo-run/tests/arguments.rs:53]: Please don't

**Comment by tomek0123456789** [comment]
Not sure if all errors should be printed to `stdout`, as it might be a breaking change?

**Review by maciektr — APPROVED** [review]
Great work!

**Pull request merged**